.github/workflows/swagger.yml

```diff
@@ -0,0 +1,28 @@
+name: Swagger
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Instead of main branch, we should build Swagger for tag. This swagger doc will be available to everyone, so it should render latest released API document (not something which is under development).
What do you think?

Reply:
yeah makes sense.. will update it